blockindex/indexbuilder_test.go

```diff
@@ -68,16 +68,14 @@ func TestIndexBuilder(t *testing.T) {
 			})
 		require.NoError(dao.Start(ctx))
 		require.NoError(indexer.Start(ctx))
-		defer func() {
-			require.NoError(indexer.Stop(ctx))
-			require.NoError(dao.Stop(ctx))
-		}()
-
 		ib := &IndexBuilder{
 			dao:     dao,
 			indexer: indexer,
 		}
-		defer ib.Stop(context.Background())
+		defer func() {
```

Comment:
why is this change needed in this PR?

Reply:
the original test is not properly written

ib is a wrapper of indexer, `ib.Stop()` calls `ib.indexer.Stop()` inside --> so `indexer.Stop()` is called twice